include/openrave/openrave.h

```diff
@@ -1613,7 +1613,10 @@ class OPENRAVE_API IkParameterization
     /// The container the iterator points to needs to have \ref GetNumberOfValues() available.
     /// Does not support custom data
     /// Don't normalize quaternions since it could hold velocity data.
-    inline void GetValues(std::vector<dReal>::iterator itvalues) const
+    ///
+    /// About I, something that behaves like std::vector<dReal>::iterator is expected.
+    template<typename I>
```

Comment:
@cielavenir can we not have single-letter variables? Thanks

Reply:
@yoshikikanemoto @eisoku9618 (because you are involved to graspingcommon!139)

@lazydroid @ntohge

I'm terribly sorry,

Would you mind telling me https://github.com/rdiankov/openrave/blob/production/include/openrave/openrave.h#L1089 `template <typename T>` meaning in the way newbies can understand? (note I'm not asking that `T is typename variable`)

Reply:
Do I need to fix all of them?

Reply:
No, just fix the new code you are making. Thanks

Reply:
this part is fine _for now_